Build.fsx

```diff
@@ -115,14 +116,14 @@ Target "TestOnly" (fun _ ->
     let nunit2TestAssemblies = !! (sprintf "Src/AutoFixture.NUnit2.*Test/bin/%s/*Test.dll" configuration)
 
     nunit2TestAssemblies
-    |> NUnit (fun p -> { p with StopOnError = false
-                                OutputFile = "NUnit2TestResult.xml" })
+    |> NUnitSequential.NUnit (fun p -> { p with StopOnError = false
+                                                OutputFile = "NUnit2TestResult.xml" })
```

Comment:
This change seems to be unrelated to the new `AppVeyor` target. Could you please commit it separately and describe its purpose in the message?